spinn_front_end_common/utility_models/reverse_ip_tag_multi_cast_source.py

```diff
@@ -240,10 +240,9 @@ def create_machine_vertex(
             resources_required,  # @UnusedVariable
             label=None, constraints=None):
         send_buffer_times = self._send_buffer_times
-        if (self._send_buffer_times is not None and
-                len(self._send_buffer_times) > 0):
-            if hasattr(self._send_buffer_times[0], "__len__"):
-                send_buffer_times = self._send_buffer_times[
+        if send_buffer_times is not None and send_buffer_times:
```

Comment:
This is still dangerous, as send_buffer_times is a user-defined value, that can be a numpy array (try changing the spike times to a numpy array in the synfire chain and you will see what I mean).

Reply:
Ugh. OK